cmd/cli/kubectl-kyverno/commands/create/permission/command_test.go

```diff
@@ -0,0 +1,193 @@
+package permission
```

Comment:
should we rename it to `role` as it generates a cluster role?

Reply:
then should i rename the command "permission" as "role" too?

Reply:
Yes, please. Or to make it clearer we can name the command `cluster-role`.